### PR TITLE
Case study part 4 complete

### PR DESCRIPTION
**Pull request by seguram10**

### CODE REVIEW

File: pacific.css [hunk 1]
```diff
@@ -0,0 +1,176 @@
+body {
+    background-color: #4F6D93;
+    color: #666666;
+    font-family: Verdana, Arial, sans-serif;
```
Comment (mbMosman) [inline pacific.css]: This should be `Arial, Helvetica, sans-serif`

File: pacific.css [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+}
+a:visited {
+    color: #344873;
+}
+a:hover {
+    color: #FFFFFF;
+}
+header, main, nav, footer, figure, figcaption, aside, section, article { display: block; }
+#mobile {
+
```
Comment (mbMosman) [inline pacific.css]: this should set display to none as this is used for "desktop"

File: pacific.css [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+}
+a:hover {
+    color: #FFFFFF;
+}
+header, main, nav, footer, figure, figcaption, aside, section, article { display: block; }
+#mobile {
+    
+}
+#desktop {
+
```
Comment (mbMosman) [inline pacific.css]: This should set display to inline

File: pacific.css [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+    padding-left: 30px;
+}
+nav ul {
+    list-style-type: none;
+    margin: 0;
+    padding-left: 0;
+    font-size: 1.2em;
+    text-align: left;
+}
+a:link {
```
Comment (mbMosman) [inline pacific.css]: You need to use a descendant selector on all of these to limit it to only the links in the nav.  The nav links should also not have the underline (but non-nav links should).

File: pacific.css [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+           background-color: #FFFFFF; }
+    #wrapper { min-width: 0; 
+               width: auto; 
+               box-shadow: none; }
+    main { margin-left: 0; }
+    nav { float: none;
+          width: auto;
+          text-align: center;
+          padding: 0.5em; }
+    nav ul { display: inline;
```
Comment (mbMosman) [inline pacific.css]: The display should be set to inline on the nav li elements.  Pay attention to which element the style needs to be applied to.  You can use the dev tools to help with this.  If you are not getting the display changes you expect, use the dev tools to try applying it to a parent or child element until the desired display effect is shown.  Then update your CSS file with the changes.

File: pacific.css [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+           padding-left: 1em;
+           padding-right: 1em;
+           font-size: 90%; }
+    h1 { font-size: 1.5em; }
+    nav { padding: 0; }
+    nav ul { display: block; 
+             margin: 0;
+             border: 2px solid #330000; } 
+    nav { display: block; }
+    #homehero { background-image: none;
```
Comment (mbMosman) [inline pacific.css]: You also need to set the display to none for the space the image took up to be gone.

File: pacific.css [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+                height: 0; }
+    #yurthero { background-image: none;
+                height: 0;}
+    #trailhero { background-image: none;
+                 height: 0;}
+    #mobile { display: inline; }
+    #desktop { display: none; }
+}
+table {
```
Comment (mbMosman) [inline pacific.css]: All of these "desktop" styles should be placed before any of the media query CSS.

File: pacific.css [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+    background-size: 100% 100%;
+    background-repeat: no-repeat;
+    height: 300px;
+    background-image: url(images/trailhero.jpg);
+    margin-left: 190px;
+}
+main {
+    background-color: #FFFFFF;
+    margin-left: 190px;
+    padding-left: 30px;
```
Comment (mbMosman) [inline pacific.css]: There should be padding on all sides of the main element.  This is why there is a blue space between the hero image and the main element.

File: pacific.css [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+    background-color: #FFFFFF;
+} 
+#wrapper { 
+    margin: auto;
+    background-color: #90C7E3;
+    width: 80%;
+    min-width: 960px; 
+    max-width: 2048px;
+    box-shadow: 3px 3px 3px #333333;
+    padding-top: 1px;
```
Comment (mbMosman) [inline pacific.css]: There should be no padding on any sides of the wrapper.  This is why there is blue space around the heading, etc.

File: pacific.css [hunk 10]
```diff
@@ -0,0 +1,176 @@
...
+    margin-left: 190px;
+    padding-left: 30px;
+}
+nav ul {
+    list-style-type: none;
+    margin: 0;
+    padding-left: 0;
+    font-size: 1.2em;
+    text-align: left;
+}
```
Comment (mbMosman) [inline pacific.css]: You need to have the list items set to display block for them to stack correctly for the desktop view.

**Comment by seguram10** [comment]
Ms. Mosman,


I've made the corrections but I am still having the same issue. Can you help me?


Best,

Monica Segura


________________________________

**Pull request closed without merging**